js/views/messageattachment.js

```diff
@@ -68,14 +68,14 @@ define(function(require) {
 			var folder = require('state').currentFolder;
 			var messageId = this.model.get('messageId');
 			var attachmentId = this.model.get('id');
-			// Loading feedback
-			this.getUI('saveToCloudButton').removeClass('icon-folder')
-				.addClass('icon-loading-small')
-				.prop('disabled', true);
 
 			var _this = this;
-			MessageController.saveAttachmentToFiles(account, folder, messageId, attachmentId)
-				.catch(console.error.bind(this)).then(function() {
+			MessageController.saveAttachmentToFiles(account, folder, messageId, attachmentId, function() {
+				// Loading feedback
+				_this.getUI('saveToCloudButton').removeClass('icon-folder')
```

Comment:
see five lines below, the code is already there 😉 For some reason it doesn't seem to work though. Try to debug and fix that instead please

Reply:
I moved it here to only show loading when user confirms directory, as above.